File: EjerciciosCpp/Actividad1/Actividad2.cpp
#include <iostream>

using namespace std;

int main(){
    int n = 0, count = 0, num = 0, sum = 0;
    cout<<"Cuantos numeros deseas sumar: ";
    cin>>n;
    while (count < n)
    {
        cout<<("Dame un numero: ");
        cin>>num;
        sum += num;
        count++;
    }
    sum = sum/n;
    cout<<"El promedio es: "<<sum<<endl;
}

File: EjerciciosPy/Actividad2.py
n = int(input("Cuantos numeros deseas sumar: "))
sum = int(0)
for i in range(n):
    num=int(input("Dame un numero: "))
    sum = sum+num
sum = sum/n
print(f"El promedio de los numeros dados es: {sum}", end="")